src/value.rs

```diff
@@ -217,10 +216,10 @@ impl Value {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+#[derive(Clone, Debug)]
```

Comment:
It'd be quite nice if we could retain these traits.

Reply:
Does `SyntaxNode` support these traits in a way we would expect?